### PR TITLE
Adds in a lemon plush toy 

### PR DESCRIPTION
**Pull request by ma44**
It should work, going to add some things and modify this a bit or something
:cl:
add: A lemon plush toy
/:cl:

### CODE REVIEW

**Comment by Supermichael777** [comment]
src.sound?

File: code/game/objects/items/toys.dm [hunk 1]
```diff
@@ -1066,6 +1067,32 @@
 	return ..()
 
 /*
+ * Lemon plushie
+ */
+ 
+ /obj/item/toy/lemonplushie
+	name = "lemon plushie"
+	desc = "A lemon plushie that feels heavier than it looks, this will make a scale unbalanced"
```
Comment (AdamElTablawy) [inline code/game/objects/items/toys.dm]: what  a shitty desc

you don't even have a period at the end

**Comment by Niknakflak** [comment]
Shitty.
Merging from master
Doesn't compile.
Not even a special sprite, it's literally the fucking lemon sprite.
No point to this toy other than you trying to milk your retarded meme that I personally killed long ago.
Does it do anything special? No, it honks and it's dumb because you are just trying to milk your meme.
And, setting a sound to a variable is un-needed, look at how other things play sounds.  They don't set the sound to a needless variable, they just reference the sound path in the playsound.

**Comment by TechnoAlchemisto** [comment]
the desc is pretty bad and seems kind of like you're patting yourself on the back with this to be quite honest

**Comment by Jalleo** [comment]
I presume this PR will get closed. @ma44 Seriously quit being so poor with your PRs and stop your salt already. Either PR some fixes or good changes.

**Pull request closed without merging**